### ISSUE TITLE
Option to configure default / home view

### ISSUE THREAD

**Issue by ryankurte** [octobox/octobox#3146]
hey thanks again for all your work on octobox! certainly makes wading through hundreds of github notifications much more pleasant ^_^

### What feature you'd like to see

the ability to filter (dependa)bot from the home page search by default.

this could be achieved in any number of ways, whether by adding a persistent `filter bots` button, setting default search parameters for the `Inbox`, defaulting to a pinned search, or splitting bot created issues / PRs out from the `Inbox`.

## Motivation

when you follow / interact with / maintain a bunch of projects the sheer volume of dependabot PRs pushes the signal-to-noise through the floor...

you can work around this by bookmarking a search with the right parameters set (`bot: false` or `-label:dependencies`), but you have to remember to click your bookmark instead of home or you end up in the robot soup again.

## Pitch

it'd be great to be able to go directly `octobox.io` and see relevant issues / immediately prioritise humans over bots ^_^

**Comment by andrew** [comment]
Agreed, I think this would be a great feature, we can leverage the existing search filters to make your ideal first view by default whilst still being able to view those other notifications by changing the search/filters. 

I’m on holiday at the moment but can look at implementing this setting in a week or so
